ytfzf

```diff
@@ -1062,6 +1062,13 @@ preview_display_image () {
 			dep_check "display" || die 3 "\nimagemagick is not installed\n"
 			killall display
 			display "$thumb_dir/${id}.jpg" ;;
+		kitty)
+			printf '\n'
```

Comment:
Since kitty takes a position to put the image, we don't need this.
Things like chafa print actual text, so we need to print a new line before printing the image.

Reply:
Right, hadn't really thought about it.